windows-container/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# build stage
+FROM myplooploops/golang:windowsservercore-1803 AS build-env
```

Comment:
probably just use the official golang image. i never tried though

Reply:
Will try it out with the official one as well.  Stay tuned!